transform.go

```diff
@@ -7,6 +7,8 @@ import (
 )
 
 const _defaultTitle = "Table of Contents"
+const _defaultTitleDepth = 1
```

Comment:
style nit: these should all be grouped -- no worry, though; gofumpt takes care of it.